Engine/source/terrain/terrCellMaterial.cpp

```diff
@@ -548,8 +548,8 @@ bool TerrainCellMaterial::_createPass( Vector<MaterialInfo*> *materials,
       // MFT_TerrainAdditive feature to lerp the
       // output normal with the previous pass.
       //
-      //if ( prePassMat )
-         //desc.setColorWrites( true, true, false, false );
+      if ( prePassMat )
+         desc.setColorWrites( true, true, true, false );
```

Comment:
Does color writes on prepass target was the responsible of the green shift we had long time ago on decals while working on fully deferred branch?
(Just to be sure before merge)